anywayproject/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for anywayproject project.
+
+Generated by 'django-admin startproject' using Django 3.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '72^tws3ivjd%lfgac%028u@dmg)$dx9#fg%psuc+ej-%2d-o-i'
```

Comment:
I don't see this used anywhere (and it should probably be loaded from the environment, or using [secrets](https://stackoverflow.com/questions/42139605/how-do-you-manage-secret-values-with-docker-compose-v3-1)

Reply:
It is created by default, and is now assigned from environment variable